org.eclipse.winery.workflowmodeler/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "workflow-modeler",
+  "version": "1.0.0",
+  "description": "A Designer For workflow",
+  "scripts": {
+    "start": "webpack-dev-server --inline --progress",
+    "test": "karma start",
+    "build": "rimraf dist && webpack --config config/webpack.prod.js --progress --profile --bail",
+    "mock": "node mock/server.js",
+    "lint": "ng lint"
+  },
+  "licenses": [
+    {
+      "type": "Apache-2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    {
+      "type": "Eclipse Public License-1.0",
+      "url": "http://www.eclipse.org/legal/epl-v10.html"
+    }
+  ],
+  "dependencies": {
+    "@angular/animations": "4.0.2",
```

Comment:
That's true. I will update to current version later.

I'd like to know when should I do this work?  Is there some good practice? For a new project, the code should be changed frequently, does it need to update version very time I commit code or just when the code is stable for a version. Thanks.